native/machine-coding/weather-app/index.js

```diff
@@ -20,7 +20,21 @@ async function checkWeather(city) {
     document.querySelector('.humidity').innerHTML = data.main.humidity + '%';
     document.querySelector('.wind').innerHTML = data.wind.speed + 'km/h';
 
-    weatherIcon.src = `images/${data.weather[0]?.main?.toLowerCase()}.png`;

```

Comment:
There is no problem with the code probably

Reply:
okay